tutorials/model_command.md

```diff
@@ -0,0 +1,271 @@
+          - ground_plane
+          - vehicle_blue
+          - vehicle_green
+```
+</details>
+
+Once you get the name of the model you want to see, you may run the following commands to get its properties.
+
+`ign model -m arg` to get the full information of the model. e.g. `ign model -m vehicle_blue`
+<details>
+  <summary>Click to see output of `ign model -m vehicle_blue`</summary>
+
+```
+      Requesting state for world [diff_drive] on service [/world/diff_drive/state]...
+
+      Name: vehicle_blue
+        - Pose: 
+            [0.000000 | 2.000000 | 0.325000]
+            [0.000000 | 0.000000 | 0.000000]
+      
+        - Link [9]
+          - Name: chassis
+          - Parent: vehicle_blue [8]
+          - Mass: [1.143950]
+          - Inertial Matrix: 
+              [0.126164 | 0.000000 | 0.000000]
+              [0.000000 | 0.416519 | 0.000000]
+              [0.000000 | 0.000000 | 0.481014]
+          - Pose: 
+              [-0.151427 | 0.000000 | 0.175000]
+              [0.000000 | 0.000000 | 0.000000]
+        - Link [12]
+          - Name: left_wheel
+          - Parent: vehicle_blue [8]
+          - Mass: [2.000000]
+          - Inertial Matrix: 
+              [0.145833 | 0.000000 | 0.000000]
+              [0.000000 | 0.145833 | 0.000000]
+              [0.000000 | 0.000000 | 0.125000]
+          - Pose: 
+              [0.554283 | 0.625029 | -0.025000]
+              [-1.570700 | 0.000000 | 0.000000]
+        - Link [15]
+          - Name: right_wheel
+          - Parent: vehicle_blue [8]
+          - Mass: [2.000000]
+          - Inertial Matrix: 
+              [0.145833 | 0.000000 | 0.000000]
+              [0.000000 | 0.145833 | 0.000000]
+              [0.000000 | 0.000000 | 0.125000]
+          - Pose: 
+              [0.554282 | -0.625029 | -0.025000]
+              [-1.570700 | 0.000000 | 0.000000]
+        - Link [18]
+          - Name: caster
+          - Parent: vehicle_blue [8]
+          - Mass: [1.000000]
+          - Inertial Matrix: 
+              [0.100000 | 0.000000 | 0.000000]
+              [0.000000 | 0.100000 | 0.000000]
+              [0.000000 | 0.000000 | 0.100000]
+          - Pose: 
+              [-0.957138 | 0.000000 | -0.125000]
+              [0.000000 | 0.000000 | 0.000000]
+        - Joint [21]
+          - Name: left_wheel_joint
+          - Parent: vehicle_blue [8]
+          - Joint type:  revolute
+          - Parent Link: [left_wheel]
+          - Child Link:  [chassis]
+        - Joint [22]
+          - Name: right_wheel_joint
+          - Parent: vehicle_blue [8]
+          - Joint type:  revolute
+          - Parent Link: [right_wheel]
+          - Child Link:  [chassis]
+        - Joint [23]
+          - Name: caster_wheel
+          - Parent: vehicle_blue [8]
+          - Joint type:  ball
+          - Parent Link: [caster]
+          - Child Link:  [chassis]
+```
+</details>
+
+
+`ign model -m arg --pose` to get the pose information. e.g. `ign model -m vehicle_blue --pose`
```

Comment:
Here, instead of `arg` use `<model_name>`.

Reply:
Addressed